Gruntfile.js

```diff
@@ -238,33 +46,31 @@ module.exports = function(grunt) {
           port: 9696
         }
       }
+    },
+
+    shell: {
+      AMDJqueryQ: {
+        command: 'npm run build:requirejs_jquery_q'
```

Comment:
nit: let's update "requirejs_jquery_q" with the same name "AMDJqueryQ" (also makes sense since we're not using requirejs anymore)